docs/CONTRIBUTING.rst

```diff
@@ -33,16 +33,13 @@ Mid and large scale issues are discussed in `Feature planning <https://github.co
 
 An issue can be developed when all the questions in the issue template are answered and there is no objection from any core member.
 
+We are using GitHub labels to show the state and attributes of the issues and Pull requests.
+If you are looking for contribution opportunities you can `Filter for these labels <https://github.com/lvgl/lvgl/labels>`__ :
 
-We are using GitHub Labels to show the state of the issue:
+- ``Simple``: Good choice to get started with LVGL contribution
+- ``PR needed``: We investigated the issue but it still needs to be implemented
+- ``Review needed``: A Pull request is opened and it needs review/testing
```

Comment:
Only showing this subset in the docs for simplicity?

Reply:
Yes, only what is really needed to make the first contribution 🙂